cmd/nginx-ingress/flags.go

```diff
@@ -19,6 +19,8 @@ const (
 	dynamicWeightChangesParam     = "weight-changes-dynamic-reload"
 	appProtectLogLevelDefault     = "fatal"
 	appProtectEnforcerAddrDefault = "127.0.0.1:50000"
+	logLevelDefault               = "info"
+	logFormatDefault              = "glog"
```

Comment:
I thought `glog` would not be the default going forward?

Reply:
this is temp for now. I will not be the default going forward but until all of the changes for struct logs are in, it will be.